libs/cli/langchain_cli/dev_scripts.py

```diff
@@ -22,10 +27,18 @@ def create_demo_server():
         mod = __import__(package["module"], fromlist=[package["attr"]])
 
         chain = getattr(mod, package["attr"])
-        add_routes(app, chain)
+        add_routes(
+            app,
+            chain,
+            config_keys=config_keys,
+        )
     except KeyError as e:
         raise KeyError("Missing fields from pyproject.toml") from e
     except ImportError as e:
         raise ImportError("Could not import module defined in pyproject.toml") from e
 
     return app
+
+
+def create_demo_server_configurable():
```

Comment:
factory has to be no-arguments unfortunately, so have to pass this as separate param